### PR TITLE
FAQ min coinjoin amount: link to stats page for current round min amount

### PR DESCRIPTION
**Pull request by MarnixCroes**
follow up https://github.com/zkSNACKs/WasabiDoc/pull/1634

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -764,7 +764,7 @@ Here's a great explanation about it:
 
 The minimum amount of bitcoin per coinjoin round is 0.00005000 BTC (5000 sats) + mining fees.
 
-> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs.
+> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs. The minimum amount for current coinjoin round(s) is displayed on the https://stats.wasabiwallet.io/ page at the _Coinjoin main net lobbies_ section. (The _Min input amount_ excludes the additionally required mining fee)
```
Comment (MarnixCroes) [inline docs/FAQ/FAQ-UseWasabi.md:767]: thought about adding it as a `:::tip`, but I think it then becomes too crowded (for such a simple question)
so added it here

Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:767]:
```suggestion
> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs.
The minimum amount for current coinjoin round(s) is displayed on the https://stats.wasabiwallet.io/ page at the _Coinjoin main net lobbies_ section. (The _Min input amount_ excludes the additionally required mining fee)
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 1 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md]: done

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -764,7 +764,8 @@ Here's a great explanation about it:
 
 The minimum amount of bitcoin per coinjoin round is 0.00005000 BTC (5000 sats) + mining fees.
 
-> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs.
+> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs. 
+The minimum amount for current coinjoin round(s) is displayed on the https://stats.wasabiwallet.io/ page at the _Coinjoin main net lobbies_ section. (The _Min input amount_ excludes the additionally required mining fee)
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:768]:
```suggestion
> The minimum amount can be higher than 5000 sats when mining fees are relatively high, to not create uneconomical outputs.
The minimum amount for the current coinjoin round(s) is displayed on the https://stats.wasabiwallet.io/ page at the _Coinjoin main net lobbies_ section. (The _Min input amount_ excludes the additionally required mining fee)
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md]: done

**Review by yahiheb — APPROVED** [review]

**Pull request merged**